src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,26 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-int totalRejected = 0;
+	private int totalRejected = 0;
 
 	public int reportRejected() {
-		return totalRejected;
+		return getTotalRejected();
 	}
 
+	@Override
 	public void countIn(int in) {
-	if (!callCheck() && in > peekaboo())
-			totalRejected++;
-			else
-				super.countIn(in);
+		if (!callCheck() && in > peekaboo())
+			setTotalRejected(getTotalRejected() + 1);
```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.